File: src/main/java/com/github/kindrat/cassandra/client/properties/StorageProperties.java
package com.github.kindrat.cassandra.client.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Data
@ConfigurationProperties(prefix = "client.storage")
public class StorageProperties {
    private String location = System.getProperty("java.io.tmpdir");
    private String propertiesFile = "connections.cfg";
}

File: src/main/kotlin/com/github/kindrat/cassandra/client/properties/StorageProperties.kt
package com.github.kindrat.cassandra.client.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.ConstructorBinding

@ConstructorBinding
@ConfigurationProperties(prefix = "client.storage")
data class StorageProperties(
        val location: String = System.getProperty("java.io.tmpdir"),
        val propertiesFile: String = "connections.cfg"
)